static/src/javascripts/lib/$.js

```diff
@@ -9,7 +9,7 @@ bonzo.aug({
     },
 });
 
-const $ = (selector: ?string | ?Node, context: ?Element | ?string): bonzo =>
+const $ = (selector: ?string | ?Node, context: ?Node | ?string): bonzo =>
```

Comment:
After some discussion with @gustavpursche, under the hood `qwery`'s executes [`querySelectorAll()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll) on the `context` argument, so perhaps this should be `?Element` after all.

@Ap0c if you're feeling saucy, you can update types in `lib/$.js` to expetc `Element` rather than `Node`. If you're not comfortable with this, or if it breaks a bunch of other modules, add the following comment and we'll go through the application fixing it more systematically:


```js
// #? Functions in this module should expect and return `Element`, not `Node`
```